rainier-tests/src/main/scala/com/stripe/rainier/core/SBCModel.scala

```diff
@@ -53,16 +53,16 @@ object SBCLogNormal extends SBCModel[Double] {
   def sbc =
     SBC(LogNormal(0, 1))((x: Real) => LogNormal(x, x))
   def goldset =
-    List(0.1320352047376991, 0.1278465305184665, 0.13045570289226482,
-      0.12809334825262858, 0.1302285851887811, 0.12971488966061975,
-      0.12858030863741665, 0.12974670510210187, 0.12864963330710052,
-      0.13046537194900923, 0.12880773413827598, 0.12985465134247504,
-      0.12844225339631635, 0.1313974020144121, 0.12744884255033428,
-      0.1311196525734243, 0.12853622813696675, 0.12853622813696675,
-      0.12977895561929287, 0.12907650766771345, 0.12983463336804282,
-      0.12873418321929223, 0.12873418321929223, 0.13013268941844386,
-      0.12923546640482494, 0.12958663528945488, 0.12958663528945488,
-      0.1300053138358873, 0.12979024604560313, 0.12983475193607927)
+    List(0.7757238193473195, 0.7757238193473195, 0.7757238193473195,
```

Comment:
I'm really confused as to what made the goldset change here, but I'm satisfied that this one is correct.